src/deviceApiCalls/schemas/setIncontextSignupDismissedAt.params.json

```diff
@@ -0,0 +1,13 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "SetIncontextSignupDismissedAt",
+    "type": "object",
+    "description": "Sets the time that the in-context Email Protection sign-up message was dismissed",
+    "additionalProperties": false,
+    "properties": {
+        "value": {
```

Comment:
Same as above. `timestamp` seems more explicit.

Reply:
@GioSensation I get an error when changing this to `timestamp`
> Value is not accepted. Valid values: "array", "boolean", "integer", "null", "number", "object", "string".